comprl/client/client_protocol.py

```diff
@@ -90,7 +90,7 @@ def error(self, msg):
         Args:
             msg (_type_): error description
         """
-        pass
+        raise Exception(msg)
```

Comment:
For now this should do it, but I intended to use this as a general error communication thing, so if the opponent loses connection we could send a message to but don't want our client to raise an exception?
Correct me if i am wrong

Reply:
I see your point. I will leave it for now until we can come up with a better solution.